src/Symfony/Component/Finder/Iterator/FilterIterator.php

```diff
@@ -12,9 +12,10 @@
 namespace Symfony\Component\Finder\Iterator;
 
 /**
- * This iterator just overrides the rewind method in order to correct a PHP bug.
+ * This iterator just overrides the rewind method in order to correct a PHP bug,
+ * which existed before version 5.5.23/5.6.7.
  *
- * @see https://bugs.php.net/bug.php?id=49104
+ * @see https://bugs.php.net/bug.php?id=68557
```

Comment:
@jakzal, it was me who added the original reference and at that time that PHP bug seemed to be related to the experienced problem. Now it turns out that the latter bug is the actual reason. The former one is still not fixed but it doesn't seem to cause any incorrect behavior.